Client.xcodeproj/project.pbxproj

```diff
@@ -1758,7 +1758,7 @@
 				F84B22541A0920C600AAB793 /* ShareTo.appex in Embed App Extensions */,
 			);
 			name = "Embed App Extensions";
-			runOnlyForDeploymentPostprocessing = 0;
+			runOnlyForDeploymentPostprocessing = 1;
```

Comment:
This is the line causing the test to fail. 
The failure is because when we try to share a page, the Fennec option is not available...
![Screenshot 2023-06-05 at 12 17 01](https://github.com/mozilla-mobile/firefox-ios/assets/1897507/fd5b62c9-b841-47e6-8f12-86e2f14b2bd1)

I have tried and without this change, the option is there and the test should pass on Bitrise